app/src/main/java/com/macbitsgoa/events/maps/MapsActivity.java

```diff
@@ -0,0 +1,161 @@
+package com.macbitsgoa.events.maps;
+
+
+import android.os.Bundle;
+import android.util.Log;
+
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.ValueEventListener;
+import com.macbitsgoa.events.R;
+
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.OnMapReadyCallback;
+import com.google.android.gms.maps.SupportMapFragment;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.MarkerOptions;
+
+import java.util.ArrayList;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.FragmentActivity;
+
+/**
+ * Google Maps Activity showing Event Markers.
+ *
+ * @author Rajath Reghunath [rrgodhorus@gmail.com]
+ */
+
+@SuppressWarnings("DuplicateStringLiteralInspection")
+public class MapsActivity extends FragmentActivity implements OnMapReadyCallback {
+
+    private static final int MAP_INIT_CAP = 15;  //initial capacity for number of venues
+    private static final String MAP_CENTRAL_LOC = "/locations/central";
```

Comment:
Are these firebase keys? If so, put all firebase keys in separate `public final class FirebaseKeys` class.